### PR TITLE
implemented object detail view

### PR DESCRIPTION
**Pull request by jsauvain**

### CODE REVIEW

File: Backend/src/main/java/com/coderbunker/supqr/SupQrApplication.java [hunk 1]
```diff
@@ -63,6 +63,7 @@ public void initialize (Bootstrap<SupQrConfiguration> bootstrap) {
 	public void run (SupQrConfiguration configuration, Environment environment) {
 		environment.jersey().setUrlPattern("/api/*");
 
+		initializeAuthentication(environment);
```
Comment (Noah-Stalder) [inline Backend/src/main/java/com/coderbunker/supqr/SupQrApplication.java:66]: cosa fai?

File: Backend/src/main/java/com/coderbunker/supqr/database/ObjectRepository.java [hunk 2]
```diff
@@ -25,10 +35,57 @@ public ObjectRepository (DSLContextFactory factory) {
...
+		Record articleRecord = getContext()
+			.select()
+			.from(ARTICLE)
+			.join(USER).onKey()
+			.where(ARTICLE.ARTICLE_ID.eq(articleId))
+			.fetchOne();
+		List<ContentTO> content = getContext()
+			.select()
+			.from(CONTENT)
+//			.leftJoin(MEDIA_CONTENT).onKey()
```
Comment (Noah-Stalder) [inline Backend/src/main/java/com/coderbunker/supqr/database/ObjectRepository.java]: was macht das grässlon?

**Review by Noah-Stalder — CHANGES_REQUESTED** [review]

File: Backend/src/main/java/com/coderbunker/supqr/SupQrApplication.java (hunk 1 above)
Comment (jsauvain) [reply Backend/src/main/java/com/coderbunker/supqr/SupQrApplication.java:66]: me gusto

Comment (Noah-Stalder) [reply Backend/src/main/java/com/coderbunker/supqr/SupQrApplication.java:66]: pas encore

**Review by Noah-Stalder — APPROVED** [review]

**Pull request merged**